roles/common/templates/etc/sshd_config

```diff
@@ -0,0 +1,92 @@
+# Package generated configuration file
```

Comment:
We add the following lines too the sshd config file:
MaxStartups 100
PrintMotd yes
PermitUserEnvironment no
MaxAuthTries 5
GatewayPorts no

and ill be adding these two in a later commit today..
Ciphers aes128-ctr,aes192-ctr,aes256-ctr,arcfour256,arcfour128,arcfour
MACs hmac-sha1,hmac-ripemd160